.sunpy-template.yml

```diff
@@ -0,0 +1,12 @@
+default_context:
+  package_name: sunkit-dem
+  module_name: sunkit_dem
+  short_description: A package for computing solar differential emission measure distributions
+  author_name: SunPy Developers
+  author_email: 
+  license: BSD 3-Clause
+  project_url: https://github.com/sunpy/sunkit-dem
+  github_repo: sunpy/sunkit-dem
+  minimum_python_version: 3.7
```

Comment:
@wtbarnes - Is 3.7 a reasonable minimum version of Python?  There might be a good use for the new data classes, so 3.7 seems reasonable to me.

Reply:
NEP 29 drops 3.6 in June anyway...